DatadogRUM/Sources/RUMConfiguration.swift

```diff
@@ -19,9 +19,9 @@ public typealias URLSessionRUMAttributesProvider = (URLRequest, URLResponse?, Da
 
 public struct RUMConfiguration {
```

Comment:
This configuration will be revisited in [RUMM-2922](https://datadoghq.atlassian.net/browse/RUMM-2922)

Reply:
Is it worth marking the ticket number in the code comment? 🤔